### PR TITLE
Adding difference in Operator capitalization for marketing vs technic…

### PR DESCRIPTION
**Pull request by bergerhoffer**
To main only

Preview: https://github.com/openshift/openshift-docs/blob/d18e381753a44986c17445ff3396565fb7446759/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization

### CODE REVIEW

**Review by skrthomas — COMMENTED** [review]
LGTM!

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -1746,6 +1746,13 @@ style guidelines. For example:
 - Node Tuning Operator
 - Cluster Version Operator
 
+[NOTE]
+====
+Brand and legal guidance for Operator names will likely differ. For marketing materials, they prefer lowercase names for anything that is not a Red Hat product.
+
+However, the Brand team recognizes that there are different standards for marketing materials versus technical content. For this reason, the title case capitalization for Operator names in technical product documentation and OperatorHub is acceptable. We are working with the Brand team to get this difference in writing somewhere more official.
```
Comment (kalexand-rh) [inline contributing_to_docs/doc_guidelines.adoc:1753]:
```suggestion
However, the Brand team recognizes that there are different standards for marketing materials versus technical content. For this reason, the title case capitalization for Operator names in technical product documentation and OperatorHub is acceptable. We are working with the Brand team to publish official guidance.
```

**Review by kalexand-rh — COMMENTED** [review]
I think this looks good!

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by GroceryBoyJr** [comment]
@bergerhoffer took a look as you requested of us. Rock On.

**Pull request merged**